Completed Step 4

xu-jiach opened the pull request:
- Created Two tables
- Modify the add_item function to post data into the datase
- Still having problems calling the conn as a global variable (will have error if I didn't call it in each function)

## What
<!--- Write the change being made with this pull request --->

## CHECKS :warning:

Please make sure you are aware of the following.

- [ ] **The changes in this PR doesn't have private information

xu-jiach committed:
Completed Step 4

- Created Two tables
- Modify the add_item function to post data into the datase
- Still having problems calling the conn as a global variable (will have error if I didn't call it in each function)

walnut07 commented on python/main.py:
```diff
@@ -22,6 +24,36 @@
     allow_headers=["*"],
 )
 
+conn = sqlite3.connect(dbpath)
```
I think this line is executed only when the server starts. In other words, the line is not executed every time it receives a http request. That's why you need to initialize the var conn in every method. I'd take the variable conn as a parameter in every method!

mutokrm reviewed (commented):
Hope this helps you debug the code!

xu-jiach committed:
Partially Finished Step 5
Could achieve functions for reading but not writing

xu-jiach committed:
Update the code

- move the db back to the parent directory
- modify the main.py correspondingly

xu-jiach committed:
Update the dockerfile

- update the correct version of dockerfile
